src/compile/unit.ts

```diff
@@ -25,6 +25,8 @@ import {parseMark} from './mark/mark';
 import initScale from './scale/init';
 import parseScaleComponent from './scale/parse';
 import {stack, StackProperties} from '../stack';
+import {SelectionDef} from '../selection';
+import {parseUnitSelection, assembleUnitSignals, assembleUnitData as assembleSelectionData, assembleUnitMarks as assembleSelectionMarks} from './selection/selection';
```

Comment:
I wonder if we could just rename `assembleUnitMarks` => `assembleUnitSelectionMarks`?  
Basically, aliasing adds another level of indirection, thus our codebase currently avoids it by just using slightly longer, but more descriptive names.